enterprise/glassfish.common/src/org/netbeans/modules/glassfish/common/ServerDetails.java

```diff
@@ -383,6 +383,17 @@ public enum ServerDetails {
         "https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/7.0.9/glassfish-7.0.9.zip", // NOI18N
         "https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/7.0.9/glassfish-7.0.9.zip", // NOI18N
         "http://www.eclipse.org/legal/epl-2.0" //NOI18N
+    ),
+    
+    /**
+     * details for an instance of GlassFish Server 7.0.10
+     */
+    GLASSFISH_SERVER_7_0_10(NbBundle.getMessage(ServerDetails.class, "STR_7010_SERVER_NAME", new Object[]{}), // NOI18N
+        GlassfishInstanceProvider.JAKARTAEE10_DEPLOYER_FRAGMENT,
+        GlassFishVersion.GF_7_0_10,
+        "https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/7.0.10/glassfish-7.0.10.zip", // NOI18N
+        "https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/7.0.10/glassfish-7.0.10.zip", // NOI18N
```

Comment:
could you switch the urls to https://repo.maven.apache.org/maven2/... ? It points to the same server but the domain is managed by apache. I will do the same in other places so that everything is standardized.

Reply:
or let me do this all in one PR, maybe easier. But you might want to update the new URL here so that it doesn't conflict

Reply:
Thanks for the update, changing the URLs.

Reply:
only the new code please :)

Reply:
Just change the links for `GF_7_0_10` ?

Reply:
i merged the PR linked above, so the old links are already changed - I hope.

If you rebase, all you have to do is to change the NEW links for 7.0.10, yes.